.github/workflows/build-test.yaml

```diff
@@ -137,6 +137,35 @@ jobs:
           push: true
 
 
+  build-kots-helm:
+    runs-on: ubuntu-20.04
+    needs: [ can-run-ci ]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: main
+          repository: replicatedhq/kots-helm
+
+      - name: Build KOTS Helm chart
+        env:
+          GIT_COMMIT: ${{ github.sha }}
+        run: |
+          curl -O -L https://raw.githubusercontent.com/replicatedhq/kots/${GIT_COMMIT}/.image.env
+          export $(cat .image.env | sed 's/#.*//g' | xargs)
+
+          export CHART_VERSION=0.0.${{ github.run_id }}-automated
+          export KOTS_VERSION=12h
```

Comment:
Why is `12h` used here?  Do we push images with this tag?

Reply:
yep, this is what all tests use